nncf/experimental/onnx/model_normalizer.py

```diff
@@ -86,16 +87,23 @@ def add_const_value_infos_to_graph(graph: onnx.GraphProto):
         nncf_logger.info('Original opset = {}'.format(model.opset_import[0].version))
         nncf_logger.info('Original ir_version = {}'.format(model.ir_version))
 
-        model.ir_version = 7  # Due to the 'Shufflenet-v1
-        modified_model = version_converter.convert_version(model, 13)
-        # ONNX shape inference
-        # https://github.com/onnx/onnx/blob/main/docs/proposals/SymbolicShapeInfProposal.md
-        modified_model = onnx.shape_inference.infer_shapes(modified_model)
-        add_input_from_initializer(modified_model)
+        try:
+            modified_model = deepcopy(model)
+            modified_model.ir_version = 7  # Due to the 'Shufflenet-v1
+            modified_model = convert_version(modified_model, 13)
 
-        onnx.checker.check_model(modified_model)
-        nncf_logger.info(
-            'Successfully converted the model to the opset = {}'.format(modified_model.opset_import[0].version))
+            # ONNX shape inference
+            # https://github.com/onnx/onnx/blob/main/docs/proposals/SymbolicShapeInfProposal.md
+            modified_model = onnx.shape_inference.infer_shapes(modified_model)
+            add_input_from_initializer(modified_model)
+
+            onnx.checker.check_model(modified_model)
+
+            nncf_logger.info(
+                'Successfully converted the model to the opset = {}'.format(modified_model.opset_import[0].version))
+        except (RuntimeError, ConvertError) as e:
+            modified_model = model
+            nncf_logger.warning(f"Failed to convert version. Use the original model. Reason: {e}")
```

Comment:
Is this ok to keep unmodified model? Will it lead to bugs during compression?

Reply:
What happen if somebody asked for per-channel quantization for a model, which isn't supported opset13?

Reply:
There will be two cases:
1. Model can be converted properly to opset13 using version converter (what our model normalizer do) `https://github.com/onnx/onnx/blob/main/docs/VersionConverter.md`.
2. Model cannot be converted to opset>=13 .

#1 can be handled, but case #2 only allows `per-tensor` quantization.